raiden_contracts/tests/test_channel_open.py

```diff
@@ -61,26 +61,29 @@ def test_open_channel_state(token_network, get_accounts):
     (A, B) = get_accounts(2)
     settle_timeout = SETTLE_TIMEOUT_MIN + 10
 
+    channel = token_network.call().getChannelInfo(1)
+    assert channel[0] == 0  # settle_block_number
+    assert channel[1] == 0  # state
+
     token_network.transact().openChannel(A, B, settle_timeout)
 
     channel = token_network.call().getChannelInfo(1)
-    assert channel[0] == settle_timeout
-    assert channel[1] == empty_address
-    assert channel[2] == 0
-
-    A_state = token_network.call().getChannelParticipantInfo(1, A)
-    assert A_state[0] is True
-    assert A_state[1] == 0
-    assert A_state[2] == 0
-    assert A_state[3] == 0
-    assert A_state[4] == fake_bytes(32)
-
-    B_state = token_network.call().getChannelParticipantInfo(1, B)
-    assert B_state[0] is True
-    assert B_state[1] == 0
-    assert B_state[2] == 0
-    assert B_state[3] == 0
-    assert B_state[4] == fake_bytes(32)
+    assert channel[0] == settle_timeout  # settle_block_number
+    assert channel[1] == 1  # state
```

Comment:
Can you also define the channel states in the tests in a common utils file used by all tests? e.g.:

```python
CHANNEL_STATE_NONEXISTENT = 0
CHANNEL_STATE_OPEN = 1
CHANNEL_STATE_CLOSED = 2
```

Reply:
done in https://github.com/raiden-network/raiden-contracts/pull/39/commits/a6fd4b5c11e52a1328331474fcc9a4c01cde13dc